github_actions/lib/dependabot/github_actions/file_parser.rb

```diff
@@ -35,6 +35,9 @@ def parse
           dependency_set += workfile_file_dependencies(file)
         end
 
+        dependencies_without_version = dependency_set.dependencies.select { |dep| dep.version.nil? }
+        raise UpdateNotPossible, dependencies_without_version.map(&:name) unless dependencies_without_version.empty?
```

Comment:
Can we use a more descriptive error message?

`UpdateNotPossible` is overloaded and opaque to our users.

Reply:
Tagging @kbukum1 since he has a list of some of our common errors that already have paved paths.